### DISCUSSION TITLE
v1.2.0🎉

### DISCUSSION THREAD

**Discussion by KRTirtho** [KRTirtho/spotube#26] [Announcements]
### New🎉
- Initial global custom reconfigurable *hotkey* support for playback controls (play-pause/next/previous)
- Credit section in the Settings page with important links
 
### Improved📈
- Macos support
- Genius (Lyrics Provider) access_token can be saved in the Login page too
- Better theme for dropdown-buttons

### Bug fixes🩹
- broken authentication IPC on Mac OS (https://github.com/KRTirtho/spotube/pull/18)
- Mac OS's global appmenu's default APP_NAME replaced with Spotube
- location of back button on macOS (https://github.com/KRTirtho/spotube/pull/21)
- windows titlebar buttons appears on Mac OS
- genius access_token not loading on initial app start

## New Contributors
* @theilgaz made their first contribution in https://github.com/KRTirtho/spotube/pull/14
* @highjeans made their first contribution in https://github.com/KRTirtho/spotube/pull/21
* @marado made their first contribution in https://github.com/KRTirtho/spotube/pull/22

**Full Changelog**: https://github.com/KRTirtho/spotube/compare/v1.1.0...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/KRTirtho/spotube/releases/tag/v1.2.0'>v1.2.0🎉</a>.</em>

**Comment by CodeVSC** [comment]
Hello! I have a question, in this project, there is a folder called android, which contains more folders called gradle and stuff... I would like to know if it is possible to compile this project to create an app for android **(apk)**.
Thank you

**Reply by highjeans** [reply]
look at #24
